app/src/main/res/layout/splash_activity.xml

```diff
@@ -5,4 +5,3 @@
   android:layout_width="match_parent"
   android:layout_height="match_parent"
   tools:context=".splash.SplashActivity"/>
```

Comment:
please revert this change.

Reply:
This was done along with changing test case in this PR. So i think it is fine for this PR that keep this test cases.